src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * List of tasks entry by user
+ */
+public class TaskList extends ArrayList<Task> {
+    public TaskList() {
+        super();
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        super(tasks);
+    }
+
+    public TaskList findTasks(TaskList tasks, String keyword) {
```

Comment:
Good naming! Named it `findTasks`, which is more appropriate, instead of `findTask`